_posts/en/newsletters/2022-12-07-newsletter.md

```diff
@@ -55,12 +55,96 @@ infrastructure projects.
+      block template; the amount of refactoring necessary was
+      equivalent to rewriting."
+  a2link="https://bitcoincore.reviews/26152#94"
+
+  q3="Why does the MiniMiner require an entire cluster? Why can’t it
+      just use the union of each transaction’s ancestor sets?"
+  a3="Some of the ancestors may already have been paid for by some of
+      their other descendants; we don't have to pay with our transaction.
+      So we need to include these other descendants in our calculations."
+  a3link="https://bitcoincore.reviews/26152#129"
+
+  q4="If transaction X has a higher _ancestor feerate_ than independent
+      transaction Y, is it possible for a miner to prioritize Y
+      over X (that is, mine Y before X)?"
+  a4="Yes. If some of Y's low-feerate ancestors have _other_ decendants that
+      are high feerate, Y doesn't need to \"pay\" for those ancestors;
+      we can, in effect, \"remove\" those ancestors from Y's ancestor set,
+      which would then increase Y's _effective_ ancestor feerate."
+  a4link="https://bitcoincore.reviews/26152#169"
+
+  q5="Can `CalculateBumpFees()` overestimate, underestimate, both, or
+      neither? By how much?"
+  a5="It will overestimate if two outputs with overlapping ancestry are
+      chosen, since they're bumped independently (without taking the shared
+      ancestry into account). Another way is if an ancestor acquires a
+      high feerate child after we've calculated the bump fees. The
+      participants couldn't think of a way bump fees could be
+      underestimated, but the question was left open."
+  a5link="https://bitcoincore.reviews/26152#194"
+
+  q6="The `MiniMiner` is given a list of UTXOs (outpoints) that the wallet
+      might be interested in spending. Given an outpoint, what are its 4
+      possible states?"
+  a6="It could be a confirmed, or unconfirmed (in the mempool), or unconfirmed
+      but already being spent by an existing transaction in the mempool,
+      or it could be an outpoint that we've never heard of."
+  a6link="https://bitcoincore.reviews/26152-2#21"
+
+  q7="What approach is taken in the \"Bump unconfirmed parent txs to target
+      feerate\" commit?"
+  a7="This commit is the main behavior change of the PR.
+      We take the information we've learned about our UTXOs and change their
+      effective values to reflect the amount of fees that go towards bumping
+      their ancestry to the target feerate. Then we run coin selection as before."
+  a7link="https://bitcoincore.reviews/26152-2#100"
+
+  q8="How does the PR handle spending unconfirmed UTXOs with overlapping ancestry?"
+  a8="After coin selection, we run the fee-bump algorithm again on the
+      _result_ of each coin selection attempt, summing the bump fees;
+      from the difference we can know that we've over-bumped due to
+      shared ancestry, and give some of the fee back to the change
+      output (increase the change value, or add a change output)."
```

Comment:
Looks pretty much correct to me. This might be slightly more precise:
```suggestion
  a8="After coin selection, we run a variant of the MiniMiner algorithm
      on the _result_ of each coin selection attempt to get an exact bump fee.
      If we have over-bumped due to shared ancestry, we can reduce
      the fees by increasing the change value if it exists, or adding a
      change output if it doesn't."
```